tests/ui/elements/list.ts

```diff
@@ -217,10 +180,11 @@ export default class List extends CustomElement {
   }
 
   /**
-   * Check if at least one item is presented in the list
+   * Check if list is empty
    */
-  async isListNotEmpty(): Promise<boolean> {
-    return this.listItems.isVisible();
+  async isListEmpty(): Promise<boolean> {
+    await this.listItems.first().waitFor({ state: 'hidden' });
+    return this.listItems.isHidden();
```

Comment:
Looks like overengineering:
.waitFor({ state: 'hidden' }) - if not, we will wait for the global timeout and just after be filed an exception, looks like redundant check in this case

Reply:
Yep, you shouldn’t synthetically create a condition when this rule becomes true. From my point of view, this method is redundant cause you can check this from a spec file directly. From your PR
` expect(await pages.catalog.resultsList.isListEmpty()).toBeTruthy();`
I believe can be changed to 
` expect(await pages.catalog.resultsList. listItems.isHidden()).toBeTruthy();`

Reply:
Will be fixed in future MR, in accordance with offline discussion.